node/core/candidate-validation/src/lib.rs

```diff
@@ -446,14 +448,40 @@ fn validate_candidate_exhaustive<B: ValidationBackend, S: SpawnNamed + 'static>(
 		return Ok(ValidationResult::Invalid(e))
 	}
 
+	let raw_validation_code = match sp_maybe_compressed_blob::decompress(
```

Comment:
nit: `validation_code` is prefixed with `raw_`, but block_data with `decompressed_`